woocommerce-services.php

```diff
@@ -1596,8 +1596,11 @@ function enqueue_wc_connect_script( $root_view, $extra_args = array() ) {
 				)
 			);
 
+			$encoded_arguments = wp_json_encode( $extra_args );
+			$escaped_arguments = function_exists( 'wc_esc_json' ) ? wc_esc_json( $encoded_arguments ) : esc_attr( $encoded_arguments );
```

Comment:
Unfortunately, `wc_esc_json` is only part of WC 3.5.5+: https://woocommerce.github.io/code-reference/namespaces/default.html#function_wc_esc_json
I am doing a fallback on the previous `esc_attr` function.

Reply:
Our support policy is [L -2](https://github.com/Automattic/woocommerce-services/issues/2131). We don't need to support WC <= 3.5.5. We should use `wc_esc_json`.

Reply:
Ah, I guess we can update this: https://github.com/Automattic/woocommerce-services/blob/trunk/woocommerce-services.php#L11

Did it in ef5d7f312b19901e98f93b279456041f9c1bcf2d , just to be safe 👍

Reply:
I think it would be good if we include a line in the changelog for the support version change.

Reply:
^ And i see there is a commit at the end for it. 👍